siriuspy/siriuspy/devices/pssofb.py

```diff
@@ -251,7 +251,7 @@ def current(self):
     @current.setter
     def current(self, value):
         """Set Current-SP vector in SOFB order."""
-        value = _np.array(value)
+        value = _np.asarray(value)
```

Comment:
@fernandohds564 , `asarray` passes a symbolic link if `value` is already a numpy object. This may corrupt the invoking context's array. I think this is dangerous and should not be done since the diff of exec time is small (367 ns vs 246 ns, for 280-size arrays). This is way I did not use `asarray` in the setter method yet.

Reply:
I changed to `asarray` because I noticed `value` was not being changed inside this method. I think we could continue using it and in the future, if the implementation changes such that `value` is changed inside the method we drop it and go back to using `array`, what do you think?

Reply:
fair.